test/pipeline_test.py

```diff
@@ -15,5 +17,42 @@ def _test():
     print(output.pandas_df)
 
 
+def test_drop():
+    pandas_df = pd.DataFrame(data={
+        'col1': [0, 1, 2, 3],
+        'col2': [3, 4, 5, 6],
+        'col3': [6, 7, 8, 9],
+        'col4': [9, 10, 11, 12]
+    })
+
+    # Drop by columns
+    try:
```

Comment:
Are the try/except blocks necessary? I we'd just let the exception `pd.testing.assert_frame_equal` bubble up.

Reply:
I think you're right. Those try except are not necessary.